### ISSUE TITLE
Late-stage functionalisation of Fenarimols with Suzuki-Miyaura Coupling. Questions

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/Series-1-Fenarimols#86]
I plan to make modification to the 4-substitution on the phenyl ring of the fenarimols to address one of the SAR gap. Just want to get some pointers on whether anyone has more experiences in  Suzuki coupling or has seen papers reporting synthetic routes for something like this:
![Untitled Wiley Document-2](https://user-images.githubusercontent.com/36176821/216498419-085da878-8443-4a72-beaf-c99617ed8d1e.png)

I found one with this procedure, but it looks like the scope doesn't cover compatibility with heteroatoms, and I'm having issues adapting this to fenarimols.
![Untitled Wiley Document-3](https://user-images.githubusercontent.com/36176821/216498648-3f25ee67-cfef-4c7a-b08c-9e7734fd90e3.png)
https://doi.org/10.1016/j.tetlet.2021.153103

**Comment by bebi78** [comment]
@fantasy121 The top reaction should work, unless the X is also a bromine or iodine. In the bottom reaction the carbamate might hydrolyse. Maybe better doing Suzuki at an earlier stage, before the ethoxycarbonylpiperazine is introduced into the molecule.

**Comment by mattodd** [comment]
Agree, but the issue really is that you're ending up with a very aromatic, greasy molecule. What's your logD looking like? Better to introduce an amine nucleophile with a Buchwald-Hartwig?

**Comment by fantasy121** [comment]
> @fantasy121 The top reaction should work, unless the X is also a bromine or iodine. In the bottom reaction the carbamate might hydrolyse. Maybe better doing Suzuki at an earlier stage, before the ethoxycarbonylpiperazine is introduced into the molecule.

Earlier stage functionalisation is a good alternative which I have considered, however I am mindful that it would take double the effort to get the same amount of compounds (time-wise, my PhD candidature is coming to an end)

I have run a few reactions using the above condition, using the intermediate leading up to the fenarimol final product. I found that the two on the left did not give me any new product, while the one on the right worked well. 
![Screenshot 2023-02-14 at 3 24 37 pm](https://user-images.githubusercontent.com/36176821/218639068-11c744d5-7fe1-45d6-a246-53f4284a6304.png)
The literature reported the reaction condition working ok with both -OH either C=O groups present. However, I noticed that their scope of exploration did not cover heterocyclic starting materials. In addition, the biuret used to complex with Pd is a urea (N-containing), so I suspect since my starting materials have pyridyl group, this might interfere with the formation of the Pd complexation, hence why pyridyl starting materials (two left ones) didn't work, but the Ph starting material (right one) worked.

**Comment by fantasy121** [comment]
> Agree, but the issue really is that you're ending up with a very aromatic, greasy molecule. What's your logD looking like? Better to introduce an amine nucleophile with a Buchwald-Hartwig?

Actually, now that you mentioned it, I approached this problem from a bad angle, I should try and functionalise the 4-substitution with something that ideally lower the LogD overall if I can help it.
The OG: LogD ~ 4.3 vs cross-coupling extension: Ph (LogD ~ 5+) and NH2 (LogD ~ 2.5)
![Screenshot 2023-02-14 at 3 41 57 pm](https://user-images.githubusercontent.com/36176821/218642166-1a0fda16-b2f5-4324-9467-6d8512e96499.png)

![Screenshot 2023-02-14 at 3 42 11 pm](https://user-images.githubusercontent.com/36176821/218643204-612e0289-1bae-4e4f-8712-1f199d1566c9.png)

![Screenshot 2023-02-14 at 3 40 31 pm](https://user-images.githubusercontent.com/36176821/218643194-f29e9b34-3491-47e7-88e5-f011d263c02c.png)

**Comment by mattodd** [comment]
Hi @fantasy121 yes N-containing heterocycles can sometimes be a problem. Marat can tell you all about that... I reckon you need an N-C cross coupling, and *very usefully* Buchwald just published on a solution to the problem that Marat was looking at. Take a look at https://pubs.acs.org/doi/10.1021/jacs.2c13520# and see whether you could use this for your functionalisation. (thanks to my student Yuhang Wang for bringing this paper to last week's group meeting!)

**Comment by bebi78** [comment]
@fantasy121 Can you also try the ketone for the left side reaction instead of the sec alcohol?

**Comment by fantasy121** [comment]
> you also try the ketone for the left side reaction instead

Yep. That'd be my next attempt

**Comment by fantasy121** [comment]
> Hi @fantasy121 yes N-containing heterocycles can sometimes be a problem. Marat can tell you all about that... I reckon you need an N-C cross coupling, and _very usefully_ Buchwald just published on a solution to the problem that Marat was looking at. Take a look at https://pubs.acs.org/doi/10.1021/jacs.2c13520# and see whether you could use this for your functionalisation. (thanks to my student Yuhang Wang for bringing this paper to last week's group meeting!)

Thanks Mat. I'll have a look at this

**Comment by bebi78** [comment]
@fantasy121 The NH2 compound might also be a good starting point for further interesting modifications, eg sulfonamides.
